go_optouts/store/memory.py

```diff
@@ -1,9 +1,31 @@
-from zope.interface import implements
-from interface import IOptOutStore
 import uuid
 
+from zope.interface import implements
+
+from .interface import IOptOutBackend, IOptOutCollection
```

Comment:
Mostly just because it's from within the same `store` package. Happy to change if you feel it's important or more consistent with what we do elsewhere?